spec.html

```diff
@@ -17760,6 +17779,121 @@ <h1>EnumerateObjectProperties ( _O_ )</h1>
             }
           </code></pre>
         </emu-note>
+        <emu-note>
+          The list of exotic objects for which hosts are not required to match CreateForInIterator was chosen because hosts historically differed in behavior for those cases, and agreed in all others.
```

Comment:
"hosts" -> "implementations and host environments". The list above has both exotic objects that an implementation must provide independent of hosts, and host-provided.

Reply:
Hm. I am happy to change to "implementations", but I think it's confusing to have "implementations and host environments": at least in my taxonomy, implementations are what provide the semantics of AOs like `EnumerateObjectProperties`, even when they are invoked on host-provided objects.

Reply:
Just "implementations" sgtm.